### PR TITLE
[release/5.0] Update Node dependencies for SignalR

### PR DESCRIPTION
**Pull request by BrennanConroy**
## Description

* Update dependencies to latest minor/patch versions

## Customer Impact

This PR includes some routine updates to the Node packages that we reference in the SignalR library. Customers can apply these updates on their own but including them as part of the package reduces the friction for using the package.

## Regression?
- [ ] Yes
- [x] No

## Risk
- [ ] High
- [ ] Medium
- [x] Low

Minor version dependency updates.

## Verification
- [ ] Manual (required)
- [x] Automated

## Packaging changes reviewed?
- [ ] Yes
- [ ] No
- [x] N/A

### CODE REVIEW

File: src/Components/Web.JS/src/webpack.config.js [hunk 1]
```diff
@@ -11,5 +11,8 @@ module.exports = (env, args) => ({
         'blazor.webassembly': './Boot.WebAssembly.ts',
         'blazor.server': './Boot.Server.ts',
     },
-    output: { path: path.join(__dirname, '/..', '/dist', args.mode == 'development' ? '/Debug' : '/Release'), filename: '[name].js' }
+    output: { path: path.join(__dirname, '/..', '/dist', args.mode == 'development' ? '/Debug' : '/Release'), filename: '[name].js' },
+    performance: {
```
Comment (captainsafia) [inline src/Components/Web.JS/src/webpack.config.js:15]: Hm -- why was this added?

Comment (halter73) [reply src/Components/Web.JS/src/webpack.config.js:15]: @BrennanConroy Can this get the same treatment as #30035 so we're not updating the TypeScript compiler and a bunch of extraneous files in a release branch?

Comment (BrennanConroy) [reply src/Components/Web.JS/src/webpack.config.js:15]: > Hm -- why was this added?

Webpack warns about assets being larger than 244Kib in size. The blazor.server.js file was already right up against that limit so any modifications could cause the warning.

Comment (BrennanConroy) [reply src/Components/Web.JS/src/webpack.config.js:15]: Changed to `maxAssetSize` so we still get warnings if the size increases again.

Comment (BrennanConroy) [reply src/Components/Web.JS/src/webpack.config.js:15]: > Can this get the same treatment as #30035 so we're not updating the TypeScript compiler and a bunch of extraneous files in a release branch?

Done

Comment (halter73) [reply src/Components/Web.JS/src/webpack.config.js:15]: `maxAssetSize` seems useful. Should we add `hints: "error"` to this and fail the build if the limit is exceeded?

If an increase in size is deemed worth it down the road we can always change the limit. I don't know if 270KiB leaves enough wiggle room for that to be a good idea considering we're already up against the default 244KiB limit.

This is all more important in the `main` branch ofc. I'm not sure we should change this at all in a release branch unless it's already breaking the build.

**Review by halter73 — APPROVED** [review]

File: src/Components/Web.JS/src/webpack.config.js (hunk 1 above)
Comment (BrennanConroy) [reply src/Components/Web.JS/src/webpack.config.js:15]: I think it warns by default, and our build fails on warnings in yarn builds.

I think the limit is fine, because if the size does increase again we'd like to know about it. Even if we just end up upping it again.
